reference/functions/html-parse.md

```diff
@@ -0,0 +1,17 @@
+# `html-parse()`
+
+The `html-parse` function accepts exactly one argument that is converted to a string. The argument is parsed as an HTML document. The function returns a DOMNodeList containing the document node at the root of the resulting tree.
+
```

Comment:
... function accepts exactly one argument. The value of the argument is converted to a string and parsed as an HTML document.

Reply:
Darfst du gerne ändern.

Reply:
ist geändert, zusammen mit ein paar weiteren kleinigkeiten.